File: Android/PhenixGroups/app/src/main/java/com/phenixrts/suite/groups/models/ChatMessage.kt
/*
 * Copyright 2019 Phenix Real Time Solutions, Inc. Confidential and Proprietary. All rights reserved.
 */

package com.phenixrts.suite.groups.models

import androidx.lifecycle.LiveData
import java.util.*

data class ChatMessage(val senderName: LiveData<String>, val message: String, val date: Date)

File: iOS/PhenixGroups/PhenixGroups/Scenes/Active Meeting Scene/Scenes/Active Meeting Chat/ChatMessage.swift
//
//  Copyright 2022 Phenix Real Time Solutions, Inc. Confidential and Proprietary. All rights reserved.
//

import Foundation
import PhenixCore

struct ChatMessage: Identifiable, Hashable {
    let id: String
    let author: String
    let text: String
    let date: Date

    init(_ rawMessage: PhenixCore.Message) {
        self.id = rawMessage.id
        self.author = rawMessage.memberName
        self.text = rawMessage.message
        self.date = rawMessage.date
    }
}
